bin/tape

```diff
@@ -25,6 +26,17 @@ opts.require.forEach(function(module) {
     }
 });
 
+if (opts.number !== false) {
+    var tape;
+    try {
+        tape = require(resolveModule("tape", { basedir: cwd }));
+    }
+    catch(err) {
+        tape = require("../"); // assume testing tape module itself
```

Comment:
can you elaborate on this code block? adding production code just for tests seems very subpar.